src/ui/views/ChainManagement/index.tsx

```diff
@@ -167,9 +167,7 @@ const ChainManagement = () => {
   };
   return (
     <div className="chain-management">
-      <PageHeader onBack={goBack}>
-        {t(connection ? 'All Chain' : 'Chain Management')}
-      </PageHeader>
+      <PageHeader onBack={goBack}>{t('All Chains')}</PageHeader>
       <ChainManagementList />
     </div>
```

Comment:
update to all chains
![hRb4Gz3Vf1](https://user-images.githubusercontent.com/12989352/148713168-bf29bea1-9bca-451b-8928-52463aa92d94.png)